bin/units

```diff
@@ -2,4 +2,12 @@
 
 set -euo pipefail
 
+# ignore any user configuration in tests
```

Comment:
this was actually unrelated but it was something which had been annoying me for some time, prior to this change having `imports.whosonfirst.importPostalcodes=false` in your local `~/pelias.json` would cause the test suit to fail 🤦‍♂️